### ISSUE TITLE
Command setScheduldedDeparture throws TypeError: tjs.setScheduledDepartureAsync is not a function

### ISSUE THREAD

**Issue by Michael-K-at-GitHub** [onokje/node-red-contrib-tesla#43]
Hi, I'm building a simple solution to turn on ScheduledDeparture for one day only (which the Tesla app cannot do).
My flow is a simple one: setScheduledDeparture on, wait for next day 11pm , setScheduledDeparture off

My problem: I can't get the Command setScheduledDeparture to work. The node throws
`TypeError: tjs.setScheduledDepartureAsync is not a function`

My setup:

- the node is configured, selected command is " setScheduledDeparture".
- I inject this message
```
{"_msgid":"c7e5c2b645ea2f52",
"payload":1644252929966,
"topic":"",
"commandArgs":{
     "enable":true,
     "departure_time":420,
     "preconditioning_enabled":true,
     "preconditioning_weekdays_only":false,
     "off_peak_charging_enabled":false,
     "off_peak_charging_weekdays_only":false,
     "end_off_peak_time":360
     }
}
```
- I also tried with only one commandArgs: "enable", still not working, same error
- I also tried with no commandArgs and get another error: "TypeError: Cannot read property 'enable' of undefined" (which is expected)
- every other tesla node (eg setChargeLimit) is working
- car is awake

What would be a working json inject? Or is this a defect?

**Comment by Michael-K-at-GitHub** [comment]
Defect confirmed by another person here: https://discourse.nodered.org/t/node-red-contrib-tesla-setscheduldeddeparture-typeerror-tjs-setscheduleddepartureasync-is-not-a-function/57942/4

**Comment by GaPhi** [comment]
Use the compatible teslajs library I have updated ***@***.***/teslajs). The
pull to the main branch is not available yet I think.

**Comment by Michael-K-at-GitHub** [comment]
Will try that. The one in src now, pulled today?

**Comment by GaPhi** [comment]
In fact, the issue is in this package dependencies I think. node-red-contrib-tesla requires "teslajs-4.10.0" to have this API available (and not 4.9.11).

**Comment by hitnrun30** [comment]
I may have a add on to this issue.  If I set the precondition to false and then to true, the time is midnight even though I sent 450.  I need to send twice to the call to get it to update.
